geo.txt

```diff
@@ -56,3 +56,443 @@ Phys 91(11):957-962
+computation and an electron diffraction study of iron diiodide. Struct Chem 22 (2):327-336 b. Hargittai M, Subbotina NY, Kolonits M, Gershikov AG (1991) Molecular structure of first-row transition
+metal dihalides from combined electron diffraction and vibrational spectroscopic analysis. J Chem Phys 94:
+7278-7286.
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Fe   Br    2.289(8)
+Br...Br    4.476(12)
+Br   Fe Br 155.9(29)
+
+# Bond lengths: re(M)(a,b)     (angstroms)
+
+Fe Br 2.276(10)
+
+NAME = ClCuH3N
+# Point Group: C3v
+# Shape: Linear --- Trigonal pyramidal
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Bittner DM, Zaleski DP, Stephens SL, Tew DP, Walker NR, Legon AC (2015) A monomeric complex of
+ammonia and cuprous chloride: H3N⋅⋅⋅CuCl isolated and characterized by rotational spectroscopy and ab initio
+calculations. J Chem Phys 142(14):144302/1-144302/10
+[http://dx.doi.org/10.1063/1.4916391]
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Cu   Cl     2.0614(7)
+Cu...N      1.9182(13)
+N    H      1.0186
+H    N...Cu 111.40(6)
+
+# Bond lengths: rs     (angstroms)
+# Bond angles:  thetas (degrees)
+
+Cu   Cl     2.058(4)
+Cu...N      1.918(4)
+H    N...Cu 111.5(1)
+
+NAME = ClHZn
+# Point Group: Coov
+# Shape: Linear
+# Bond lengths: r0     (angstroms)
+# Source Paper: Pulliam RL, Sun M, Flory MA, Ziurys LM (2009) The sub-millimeter and Fourier transform microwave
+spectrum of HZnCl (X1 Σ+). J Mol Spectrosc 257(2):128-132
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+H  Zn 1.519
+Zn Cl 2.083
+
+# Bond lengths: rs     (angstroms)
+
+H  Zn 1.519
+Zn Cl 2.082
+
+# Bond lengths: rm     (angstroms)
+
+H  Zn 1.5050(17)
+Zn Cl 2.08293(21)
+
+NAME = ClH3Si
+# Point Group: C3v
+# Shape: Tetrahedral
+# Bond lengths: re(se)     (angstroms)
+# Bond angles:  thetae(se) (degrees)
+# Source Paper: Cazzoli G, Puzzarini C, Gauss J (2012) Rotational spectrum of silyl chloride: hyperfine structure and equilibrium
+geometry. Mol Phys 110(19-20):2359-2369
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Si Cl   2.04584(6)
+Si H    1.46878(16)
+Cl Si H 108.427(13)
+
+NAME = Cl2OS
+# Point Group: Cs
+# Shape: Trigonal pyramidal
+# Bond lengths:     r0     (angstroms)
+# Bond angles:      theta0 (degrees)
+# Dihedral angles:  tau0   (degrees)
+# Source Paper: Martin-Drumel MA, Roucou A, Brown GG, Thorwirth S, Pirali O, Mouret G, Hindle F, McCarthy MC, Cuisset
+A (2016) High resolution spectroscopy of six SOCl2 isotopologues from the microwave to the far-infrared. J
+Chem Phys 144(8):084305/1-084305/7
+[http://dx.doi.org/10.1063/1.4942024]
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+S=O         1.434(9)
+S Cl        2.072(3)
+Cl S=O      108.00(6)
+Cl S Cl     97.08
+Cl S=O...Cl 104.0(3) 
+
+# Bond lengths:     re     (angstroms)
+# Bond angles:      thetae (degrees)
+# Dihedral angles:  taue   (degrees)
+
+S=O         1.4340(7)
+S Cl        2.0700(3)
```

Comment:
It would be nice, for aesthetics and easy of reading, if all columns of atoms were aligned, as in the examples that I currently have in the main fork. I guess this means that double-bonds would need to be made a bit longer: S==O (for example).  Have you considered what notation to use for triplet bonds? The Chromium dimer (Cr2) has a sextuple bond, so I'm not sure how that would be denoted either 😂

Reply:
Yes, I can see how we will have an alignment problem with symbols like "...", "=" and elements with a different number of characters in their name.  I have two proposals. 
(1) We can leave a fixed amount of spacing in our columns each time (this will also standardize our representation across different molecules) i.e. ##***##***## where ## represents element names (supposing we don't have any of the newer elements) and *** represents how they are bonded. Single bonds are default to three spaces, double bonds " = ", triple bonds " ≡ " and the intermolecular forces are represented by "...". i.e.

```suggestion
S  = O           1.434(9)
S    Cl          2.072(3)
Cl   S  = O      108.00(6)
Cl   S    Cl     97.08
Cl   S  = O...Cl 104.0(3) 
```

Reply:
If you think we'll end up taking too long on fomatting using (1) then
(2) we can keep our previous convention but we use (2) to denote a double bond, (3) for a triple bond etc. i.e.
```
S O (2)     1.4340(7)
S Cl        2.0700(3)
Cl S=O      107.703(6)
Cl S Cl     96.929
Cl S=O...Cl 103.58(3)
```

Reply:
Thanks for making these proposals!
I also (along with you) prefer the first one. 
For sextuple bonds like in the Cr2 molecule, we can maybe have something like:
```
Cr-6-Cr
```
to denote that the bond order is 6, but it's not really crucial (for our purposes) to know the order of a bond, especially when it gets to such a high bond order.